src/config/strings.js

```diff
@@ -69,7 +69,8 @@ export default {
       maxBtnTxt: 'Max',
       balanceTitleTxt: 'Available balance',
       inputPlaceholderTxt: '0.00',
-      transferBtnTxt: 'Transfer'
+      transferBtnTxt: 'Transfer',
+      connectWalletBtnTxt: 'Connect Wallet'
```

Comment:
If you read the component `LoginWalletButton` so stick with this prefix also here, or change the component name to `ConnectWalletButton`